datastore/src/main/scala/magnolify/datastore/EntityType.scala

```diff
@@ -24,14 +24,16 @@ import magnolify.shared.{CaseMapper, Converter}
 import magnolify.shims.FactoryCompat
 import magnolify.shims.JavaConverters._
 
-import scala.annotation.implicitNotFound
+import scala.annotation.{implicitNotFound, StaticAnnotation}
 import scala.language.experimental.macros
 
 sealed trait EntityType[T] extends Converter[T, Entity, Entity.Builder] {
   def apply(v: Entity): T = from(v)
   def apply(v: T): Entity = to(v).build()
 }
 
+class excludeFromIndexes(val exclude: Boolean) extends StaticAnnotation with Serializable
```

Comment:
Should we make `exclude: Boolean` default to true?

Reply:
Great idea! Force-pushed the update 🙏